### ISSUE TITLE
Crash with Wayland native execution on 2021.12.2

### ISSUE THREAD

**Issue by 9Lukas5** [jitsi/jitsi-meet-electron#681]
## Description

If running Jitsi in Wayland native it crashes on the newest version(s). Maybe introduced by a newer electron version?
In Version 2021.11.2 it's working fine.

I also discovered it happens not always, but most of the time.

---

## Current behavior

The app crashes on a memory access attempt

---

## Expected Behavior

Application starting

---

## Possible Solution
---

## Steps to reproduce

Run application in a wayland environment natively:  
```
<path to bin> --enable-features=UseOzonePlatform,WebRTCPipeWireCapturer --ozone-platform=wayland
```

## Terminal Output

```log
14:07:12.177 › Checking for update
[25346:1204/140712.186985:ERROR:gpu_init.cc(457)] Passthrough is not supported, GL is egl, ANGLE is 
[25346:1204/140712.221654:ERROR:sandbox_linux.cc(376)] InitializeSandbox() called with multiple threads in process gpu-process.
[25310:1204/140712.228792:ERROR:cursor_loader.cc(114)] Failed to load a platform cursor of type kNull
/tmp/.mount_jitsi-sPs12T/jitsi-meet: Zeile 2: 25310 Speicherzugriffsfehler  (Speicherabzug geschrieben) "${BASH_SOURCE%/*}"/jitsi-meet.bin --no-sandbox "$@"
```

---

# Environment details

Ubuntu 20.04 Focal Fossa
Gnome 41 on Wayland
Jitsi 2021.12.2

---

**Comment by saghul** [comment]
Can you provide a backtrace? What does the console show when you start it from a terminal?

**Comment by 9Lukas5** [comment]
Oh, dumb me, totally forgot to add that, sorry. Added it to the OP :+1:

**Comment by saghul** [comment]
Thanks! Looks like an Electron but, someone posted the same backtrace on .1.

**Comment by csett86** [comment]
With the now unnecessary debian sandbox fix (and thus the wrapper script) removed (#684), it at least starts, but screenshare still loops and crashes (like described in #677)

**Comment by vchernin** [comment]
Note you shouldn't need `--enable-features=UseOzonePlatform` regardless since that should be on by default now (at least in recent Chromium).
